### PR TITLE
first omp_places cache test iteration

### PR DESCRIPTION
**Pull request by mjcarr458**

### CODE REVIEW

**Comment by mjcarr458** [comment]
Here is a very basic test that just sets the value of omp_places to ll_caches. I can't think of a way to actually confirm the cores are sharing a common cache as implied in the spec, particularly making it system agnostic as well. If anyone has a idea on how to implement that, it would be greatly appreciated.

File: tests/5.1/env_var/test_omp_places_env_ll_caches.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include <omp.h>
```
Comment (nolanbaker31) [inline tests/5.1/env_var/test_omp_places_env_ll_caches.c:1]: Needs header w/ file name & test description https://crpl.cis.udel.edu/ompvvsollve/documentation/codingstandards/

**Review by spophale — APPROVED** [review]

**Review by jrreap — APPROVED** [review]
Looks good to go!

File: tests/5.1/env_var/test_omp_places_env_ll_caches.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
+//-----------------test_omp_places_ll_caches.c---------------//
```
Comment (krishols) [inline tests/5.1/env_var/test_omp_places_env_ll_caches.c:1]: Very minor, but test name does not match file name

**Pull request merged**